src/web.md

```diff
@@ -83,6 +83,15 @@ The following resources can help you get started:
 * You can chat and ask web-related questions on the
   **#help** channel on [Discord][].
 
+---
+
+<sup>*</sup> We don’t recommend using Flutter to build static
+  content websites with a lot of text and images such as blogs,
+  event pages, and news sites. For these types of sites,
+  it's best to use traditional web approaches,
+  but you _can_ use Flutter to embed interactive experiences
+  into these websites.
+
```

Comment:
This feels duplicative of the text below, which falls just after the list of scenarios?

> Not every HTML scenario is ideally suited for Flutter at this time. For example, text-rich flow-based content such as blog articles benefit from the document-centric model that the web is built around, rather than the app-centric services that a UI framework like Flutter can deliver.